src/index.html

```diff
@@ -5,10 +5,42 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header">
+        <a href="#home">
+          <img
+          class="logo"
+          src="./images/logo.png"
+          alt="moyo logo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav-list">
+            <li class="nav-item"><a class="nav-link  is-active" href="#">Apple</a></li>
```

Comment:
```suggestion
            <li class="nav-item">
              <a class="nav-link  is-active" href="#apple">Apple</a>
            </li>
```

let's make it more readable
fix in all places